wetmap/src/components/newModals/shopModal/view.tsx

```diff
@@ -17,54 +16,23 @@ import style from './style.module.scss';
 import defaultHeaderPicture from '../../../images/blackManta.png';
 import ButtonIcon from '../../reusables/buttonIcon';
 import Icon from '../../../icons/Icon';
+import { DiveShop } from '../../../entities/diveShop';
 
 
 type ShopModelViewProps = {
```

Comment:
Seems like you don't have eslint configured. Please make sure you have eslint extension for vscode and it's configured as a formatter for this project.

Reply:
Thanks!